web/bfex/components/data_pipeline/tasks/keywords.py

```diff
@@ -0,0 +1,73 @@
+from bfex.components.data_pipeline.tasks import Task
+from bfex.components.scraper.scrapp import Scrapp
+from bfex.models import *
+from bfex.common.exceptions import WorkflowException
+from bfex.components.key_generation.rake_approach import *
+
+
+class UpdateKeywordsFromScrape(Task):
+    """
+    Updates Keywords of a Faculty Members data in elastic.
+    """
+    def __init__(self):
+        self.task_name = "Update Keywords From Scrape"
+
+    def is_requirement_satisfied(self, data):
+        satisfied = True
+
+        if (not isinstance(data, tuple) or
+                not isinstance(data[0], str) or
+                not isinstance(data[1], Scrapp)):
+            satisfied = False
+
+        return satisfied
+
+
+    def run(self,data):
+       # professor_id = data[0]
+        #scrapp = data[1]
+        for faculty in data:
+            print(faculty.faculty_id)
+            professor_id = faculty.faculty_id
+            print(professor_id)
+
+        search_results = Faculty.search().query('match', faculty_id=professor_id).execute()
```

Comment:
Searching by the id is different than searching for text. You can use the safe_get method defined in models to do this in a cleaner way. No sense searching through all faculty when elastic can easily get the one we want based on the unique id. Take a look at the data ingestion blueprint for an example.